cfpb/cfgov-refresh
# DF-1347_Adding_Pagination_Validation

- Add better pagination validation

## Changes
- Updated _includes/post-macros.html to add notification class. 
- Added ' _includes/post-macros.html' to add notification logic.

## Test
- Navigate to 'http://localhost:7000/blog ' using Safari.
- Enter 't' in the pagination input field and click 'Go'
- You should see the notification message.

## Notes 
- @jimmynotjim , not sure if inserting container was best solution but was only way
to get notifier inserted.
- Updated markup to add notification below pagination form.

## Review 
- @jimmynotjim  
- @anselmbradford 
- @Scotchester  
- @KimberlyMunoz 

## Screenshots

Mobile:
![screen shot 2015-05-22 at 12 14 48 pm](https://cloud.githubusercontent.com/assets/1696212/7774836/2d42c6ca-007c-11e5-9375-d2e45de7bf6a.png)

Tablet:
![screen shot 2015-05-22 at 12 15 10 pm](https://cloud.githubusercontent.com/assets/1696212/7774844/3913411e-007c-11e5-93eb-cd40daf5f9f8.png)

Desktop:
![screen shot 2015-05-22 at 12 15 30 pm](https://cloud.githubusercontent.com/assets/1696212/7774852/44733758-007c-11e5-8ff0-7df2091f1add.png)
